### ISSUE TITLE
Printing results in pure black stars no matter the value.

### ISSUE THREAD

**Issue by matthewdaize** [kartik-v/bootstrap-star-rating#47]
When the user attempts to print the page with star rating on it (eg. a summary report), the stars come out as solid black.

Is there a workaround for this?

**Comment by kartik-v** [comment]
- option 1 - color print :-)
- option 2 - you can try to override the css to use a lighter grey color for the empty star (do not use white as star will be totally hidden).

**Comment by matthewdaize** [comment]
Ah. After some digging, it turns out the browser overrides all CSS values and strips out color no matter what...

You cannot override it using a standard stylesheet and the parameter to print background/color does not work.

The only solution I can find (and it might not be feasible) would be to replace the symbols with empty star instead of full - might need to look to another option too.

**Comment by kartik-v** [comment]
I am not sure how you are setting the CSS... here is an example of what you should be doing...

- set `containerClass` property in the plugin to your custom CSS value - e.g. `rating-custom`
- then set a custom css file for yours named `custom.css` and add CSS for the star color this way
```css
.rating-custom.rating-stars {
     color: #fde16d; /* change this to your color */
}
```
- load the `custom.css` at the end after all the other css

**Comment by martynling** [comment]
@matthewdaize I encountered the same problem that you've had. I tried @kartik-v 's suggestion which seemed like it would work but ultimately only enabled me to change the screen presentation. For some obscure reason, when it came to printing the CSS was being overridden. Anyway, I found a working solution, which, in the absence of a better solution, is good enough for me to enable my client to print off reports in the morning. I place this in the head of the page once everything else has loaded:

```<style>
    @media print {
        /* Make the background stars disappear */
        .rating-container,
        .rating-container:before {
            color: rgba(0, 0, 0, 0)!important;
        }

        .rating-stars {
            color: #fde16d;
        }
    }
</style>```
